test/app/dftb+/hybrid/periodic/Acene-gamma-matrix-force-cam/dftb_in.hsd

```diff
@@ -0,0 +1,61 @@
+Geometry = GenFormat {
+   <<< Acene_rattled.gen
+}
+
+Hamiltonian = DFTB {
+
+  SCC = Yes
+  MaxSccIterations = 100
+  SccTolerance = 1e-12
+
+  Filling = Fermi {
+    Temperature [K] = 0.0
+  }
+
+  MaxAngularMomentum = {
+    C = "p"
+    H = "s"
+  }
+
+  SlaterKosterFiles = Type2FileNames {
+    Prefix = "slakos/origin/hybrid/cam/HC/"
+    Separator = "-"
+    Suffix = ".skf"
+  }
+
+  Hybrid = CAM {
+    CoulombMatrix = Truncated {
+      CoulombCutoff = 2.885010
+      GSummationCutoff = 50.985000
+    }
+    Screening = MatrixBased {}
+  }
+
+  KPointsAndWeights = SupercellFolding {
```

Comment:
I blocked everything except for the supercell folding technique, so this would actually result in a (useful) error message. Not nice thought, since both ways are in principle equivalent, I just spared some additional logic by that choice.

Reply:
I have now added some additional logic to accept any k-point specification that leads to a Gamma-only sampling.